### ISSUE TITLE
Field options missing

### ISSUE THREAD

**Issue by Rodrigueztb** [TabularEditor/TabularEditor#1088]
Field options is missing (last, first, average, min, max, etc) in power bi after adding a calculation group in tabular editor. 

(https://learn.microsoft.com/pt-br/power-bi/create-reports/media/service-aggregates/power-bi-aggregate-average.png for example this menu options)

When i use tabular editor and add a calculation group (with information or empty), some field option is missing

**To Reproduce**

1. Go to Tabular editor
2. Click on tables, create new, calculation group
3. Save

Delete the changes dont fix the problem.

**Application specifics**
 - Tabular Editor 2.17.3
 - Semantic Engine used: Power BI Desktop 2.115.1006.0 64-bit

**Comment by otykier** [comment]
Duplicate of #871, #636, #628, #614, #544, ...

TLDR: Calculation Groups requires the `DiscourageImplicitMeasures` property on the model to be set to `true`. This disables implicit columns (i.e. the ability to summarize columns in visuals). You'll have to either remove the calc groups, and set `DiscourageImplicitMeasures` back to `false`, or create explicit measures for any column you intend to summarize in a visual.
